### ISSUE TITLE
Unwanted values if two wg interfaces present

### ISSUE THREAD

**Issue by sc0rp10** [MindFlavor/prometheus_wireguard_exporter#16]
Hi,
I have two wg interfaces on the same node (wg0/wg1) and prometheus_wireguard_exporter.
I run two exporter instances using two different ports and configs:
```
./prometheus_wireguard_exporter -p 11111 -n /etc/wireguard/wg0.conf
./prometheus_wireguard_exporter -p 22222 -n /etc/wireguard/wg1.conf
```

Everything works fine but I got duplicated metrics with different friendly_names:
```
# instance #1 with wg0.conf
wireguard_sent_bytes_total{interface="wg1",public_key="xxx",friendly_name="foo"} 0
wireguard_sent_bytes_total{interface="wg0",public_key="yyy"} 0
wireguard_sent_bytes_total{interface="wg0",public_key="xxx"} 0

# instance #2 with wg1.conf
wireguard_sent_bytes_total{interface="wg1",public_key="xxx"} 0
wireguard_sent_bytes_total{interface="wg0",public_key="yyy",friendly_name="test"} 0
wireguard_sent_bytes_total{interface="wg0",public_key="xxx",friendly_name="test2"} 0
```

I think there's a problem with stats command: instead of using `wh show all dump` exporter should use specified interface name like `wg show wg1 dump` if `extract_names_config_file` was present
```

**Comment by MindFlavor** [comment]
It's a good idea. I'll give it a try as soon as possible (I'm a bit swamped these days :crying_cat_face: )

**Comment by MindFlavor** [comment]
This should be addressed in 3.2. Please let me know if it's better!
